### PR TITLE
Added some doc to README

### PR DESCRIPTION
**Pull request by xbill82**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+Client is will manage all the communications for you. You'll just have to
+concentrate on the data you need.
+
+To start working with Tapas, create a new instance of `Tapas`, providing the
+name of the transport protocol and the address of the endpoint. The supported
+transport protocols are `http` and `websocket`. 
 
 ```js
-let tapas = new Tapas('websocket', {port: 8080})
+let tapas = new Tapas('websocket', {endpoint: 'http://localhost:7513'})
```
Comment (AnthonySendra) [inline README.md]: `websocket` looks like an option. We probably have to put this at the end of the function? Something like this:
`let tapas = new Tapas('localhost:7513', 'websocket')` or directly `let tapas = new Tapas('ws://localhost:7513')`. I don't know

File: README.md [hunk 2]
```diff
@@ -37,63 +62,105 @@ tapas
...
-  .need({users: {name: undefined}})
-  .with(<where, order, size, from, anything to pass to the server>)
-  .execute()
-  .then((res) => console.log(res.users))
+  .need({
+    users: {
+      name: undefined,
+      age: 26
+    },
+    meta: [count]
```
Comment (AnthonySendra) [inline README.md]: `meta: ['count']`

**Review by AnthonySendra — COMMENTED** [review]
I'm wondering about the "dreaming example" at the top of readme. Something like:

```
tapas
  .need({user: {name: undefined}})
  .subscribe()
  .forEach((res) => console.log(res.user.name))
```

File: README.md (hunk 1 above)
Comment (xbill82) [reply README.md]: I like `'ws://localhost:7513'`

File: README.md (hunk 2 above)
Comment (xbill82) [reply README.md]: yep!

**Pull request merged**